tests/hikari/test_guilds.py

```diff
@@ -239,6 +239,11 @@ def test_format_avatar(self, model, mock_user):
         mock_user.format_avatar.assert_called_once_with(ext="png", size=4096)
         assert result is mock_user.format_avatar.return_value
 
+    async def test_ban(self, model, mock_user):
+        result = await mock_user.ban(reason="cuz u was meen")
+        mock_user.ban.assert_awaited_once_with(reason="cuz u was meen")
+        assert result is mock_user.ban.return_value
+
```

Comment:
This test is wrong, you are mocking `ban` and then just checking that you are calling it.

You should not mock ban and check if it calls `app.rest.ban_user` with the correct args

Reply:
I don't understand what u mean. What do I have to put on the test? I'm pretty new to pytest and testing in general. So if you could tell me literally what to do, I could get some ideas